pkg/controller/node/node_controller.go

```diff
@@ -142,5 +147,40 @@ type ReconcileNode struct {
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileNode) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	return r.autoscaler.Reconcile(request)
+
+	res, err := r.autoscaler.Reconcile(request)
+	if err != nil {
+		return res, err
+	}
+
+	// Fetch the RouteReflector instance(s)
+	routereflector := &routereflectorv1.RouteReflector{}
+	err = r.client.Get(context.Background(), client.ObjectKey{
+		Namespace: routeReflectorConfigNameSpace,
+		Name:      routeReflectorConfigName,
+	}, routereflector)
+
+	if err != nil {
+		if errors.IsNotFound(err) {
+			// Request object not found, could have been deleted after reconcile request.
+			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
+			// Return and don't requeue
+			return reconcile.Result{}, nil
```

Comment:
If user doesn't create the CR or deletes later the whole migration will stop. Maybe this is a special CR and if isn't existst the operator has to create it

Reply:
I agree, the operator should handle the missing the CR. On the other hand that could be part of the deployment process in a nicer way.